plugin/pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -751,13 +806,25 @@ func benchmarkExpire(b *testing.B, podNum int) {
 	}
 }
 
-func makeBasePod(nodeName, objName, cpu, mem string, ports []v1.ContainerPort) *v1.Pod {
+type testingMode interface {
+	Fatalf(format string, args ...interface{})
+}
+
+func makeBasePod(t testingMode, nodeName, objName, cpu, mem, oir string, ports []v1.ContainerPort) *v1.Pod {
 	req := v1.ResourceList{}
 	if cpu != "" {
 		req = v1.ResourceList{
 			v1.ResourceCPU:    resource.MustParse(cpu),
 			v1.ResourceMemory: resource.MustParse(mem),
 		}
+		if oir != "" {
+			if len(strings.Split(oir, ":")) != 2 {
+				t.Fatalf("Invalid OIR string: %v", oir)
```

Comment:
@k82cn wondering what is the merit of using `panic` over `t.Fatalf`? 
`t.Fatalf` also exits the test case immediately and gracefully too(unlike panic). Also `t.Fatalf` looks consistent with existing `cache_test.go` as well, which does not have `panic` used anywhere.